### DISCUSSION TITLE
Suggesting an alternative translation for "Specificity"

### DISCUSSION THREAD

**Discussion by snrazavi** [snrazavi/Persian-AI-and-Machine-Learning-Glossary#4] [Suggested terms and/or translations]
## Suggestion
A better translation for **Specificity** which is more accurate and more clear would be: ...

This is a better technical translation because...

**Comment by sparsaiyan** [comment]
مشخصه ی انحصاری
مشخصه ی منحصر به فرد

**Reply by snrazavi** [reply]
Thanks for your suggestion! It's a really tricky term. It is related to sensitivity and I am looking for a translation which is shorter, more clear and more accurate. Something like:
خاص بودن؛ میزان اختصاصی بودن

But they are long as well and don't convey the exact meaning.

Please don't hesitate to provide additional suggestions.
